src/assets/variables.scss

```diff
@@ -0,0 +1,16 @@
+@import "./fonts";
+
+//fonts
+$text-login-regular: normal 18px "Montserrat";
+$text-login-bold: 700 36px "Montserrat";
+$text-error-message-semibold: 600 14px "Montserrat";
+$text-login-button-bold: 700 18px "Montserrat";
+
+//colors
+$loginWelcomeBlock: #007eff;
```

Comment:
Same here, change name or declare in another place
Actually, you could declare all used colors here, because i think we will have one color theme
Like blue: ..., grey: ...